flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ActiveResourceManager.java

```diff
@@ -18,6 +18,7 @@
 
```

Comment:
This commit should be prior to 7d2ded.

Reply:
It is.
Github messed it up. It shows commits ordered by the committing time (the time in the commit message), instead of the order in the code base.